v3/plugins/dbaeumer/vscode-eslint/2.1.1/meta.yaml

```diff
@@ -0,0 +1,15 @@
+apiVersion: v2
+publisher: dbaeumer
+name: vscode-eslint
+version: 2.1.1
+type: VS Code extension
+displayName: ESLint
+title: ESLint JavaScript
+description: Integrates ESLint JavaScript into Che-Theia.
+icon: https://open-vsx.org/api/dbaeumer/vscode-eslint/2.1.1/file/eslint_icon.png
+repository: https://github.com/microsoft/vscode-eslint
+category: Other
+firstPublicationDate: '2020-11-13'
+spec:
+  extensions:
+  - https://open-vsx.org/api/dbaeumer/vscode-eslint/2.1.1/file/dbaeumer.vscode-eslint-2.1.1.vsix
```

Comment:
Because we need to contribute this plugin to CRW, let's use the built-one. I've built version 2.1.1 here: https://download.jboss.org/jbosstools/vscode/3rdparty/vscode-eslint/vscode-eslint-2.1.1-1e15d3.vsix

Reply:
Done